src/ApexCharts.component.js

```diff
@@ -54,7 +54,7 @@ export default {
       });
     });
   },
-  beforeDestroy() {
+  beforeUnmount() {
```

Comment:
isn't beforeUnmount vue3 hook? What if the user is using vue2?

Reply:
It's fix for vue2, like in example: https://codepen.io/uakniow/pen/NWvoqEv
beforeDestroy is to late to   this.chart.destroy(); 
If You use group apexjs crash, see bug #392